### ISSUE TITLE
Altered Carbon - Richard K Morgan

### ISSUE THREAD

**Issue by bryoncodes** [ElijahCaine/book_club#15]
http://www.amazon.com/Altered-Carbon-Takeshi-Kovacs-Novels/dp/0345457684

**Comment by bryoncodes** [comment]
Apparently, I don't know how to to tag this as "book suggestion"

**Comment by pop** [comment]
Yeah, I made bad instructions. I'll tag it as a book suggestion. The repo isn't open enough for public tagging.
